e2e-tests/cypress/integration/guidance-page-when-you-can-appeal.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Guidance Page - Before you Appeal
+  As a PO on the appeal service
+  I want prospective appellants to be aware of who the service is for
+  So that only those who need to appeal through the new appeal service do so
+
+  Scenario Outline: AC1: Appellant selects <page> page from the Content list
+    Given the appellant is on the when you can appeal page
```

Comment:
This doesn't tell me whether appellant has arrived on this page by clicking on Next button or by clicking on the link in the content list.

Reply:
Does it need to? If so this is a different test needing different acceptance criteria

Reply:
There is nothing wrong but an observation that this page can be accessed in 2 ways and will you be covering both the ways?